website/blog/flink-native-support/index.mdx

```diff
@@ -0,0 +1,83 @@
+A metadata can be standardized as well as enriched with freestyle facets similar to an approach taken by OpenLineage.
+The proposal defines a generic data model for all the Flink’s connectors.
+Based on the sources and sinks metadata, Flink creates a lineage graph which also works well for Flink SQL that was not supported so far.
+
+
+The lineage graph is exposed via a brand new job status change listener that can be implemented by third parties.
+
+As a result, communities like OpenLineage can prepare their listener implementation and consume lineage metadata exposed natively through developers API.
+Inspired or not, Flink lineage interfaces look pretty similar to OpenLineage data model which makes the translation straightforward.
+
+
+### Towards native lineage support in Flink
+
+The major complexity of implementing this is that it requires writing code in separate repositories with different release cadence.
+
+The first step taken was to create lineage interfaces’ definitions in the Flink core repository, which was released in Flink 1.19.
+Flink core has also been extended with a mechanism to create lineage graphs out of the sinks’ and sources’ lineage interfaces as well as expose the graph through the job listener.
+This has already been merged to Flink’s main branch and will be released in Flink 2.0.
+
+Next step is to implement lineage for the sources and sinks which oftentimes in separate repositories.
+This has been already done for [flink-connector-kafka](https://github.com/apache/flink-connector-kafka) within this [PR](https://github.com/apache/flink-connector-kafka/pull/130/files)
+as a reference implementation which serves also as a proof-of-concept for lineage listeners and end to end test verifying all the components put together produce the expected outcome.
+
+At the same time, OpenLineage community was working on the job listener (see this [PR](https://github.com/OpenLineage/OpenLineage/pull/3099)).
+Although the PR is not merged at the moment of writing this post, it already answered some important questions.
+
+Is Flink lineage data model convertible to OpenLineage model? Yes it is, although the answer to this is not straightforward given Flink jobs can read Kafka topics based on regex patterns, while OpenLineage expects each topic being identified as a separate dataset.
+
+Does the approach allow extracting lineage from Flink SQL?
+Yes, we were able to run an end-to-end test which executes SQL query on Kafka topics and verify the listener produces proper OpenLineage events for this scenario.
+
+![Code diagram](./diagram.png)
+
+This is where we are now. The next implementation steps are:
+
+ * Implement lineage interfaces for other Flink connectors.
+ * Release new OpenLineage listener.
+
+We also need to wait for the Flink community to release Flink 2.0.
+Further work is required to design and implement column level lineage for Flink SQL.
```

Comment:
for the sake of consistency I would go for 

 * Implement lineage interfaces for other Flink connectors.
 * Release new OpenLineage listener.
 * Wait for the Flink community to release Flink 2.0.
 * Design and implement column level lineage for Flink SQL.
 
 or at lease not mix passive and active in last two